src/utils/account/refreshAccount.tsx

```diff
@@ -29,14 +30,18 @@ const setNewAccount = async () => {
 };
 
 export async function refreshAccount() {
-  const provider = getAccountProvider();
+  const provider = getAccountProvider() as IDappProvider;
   if (provider == null) {
     throw 'Provider not initialized';
   }
-  if (provider.isInitialized()) {
+  if (!provider.isInitialized || provider.isInitialized()) {
```

Comment:
why first condition?

Reply:
because some providers do not have `isInitialized` method, but at the same time, if the `isInitialized` method is missing we need to set up a new account.